app/services/user_session_context.rb

```diff
@@ -1,18 +1,19 @@
 class UserSessionContext
   DEFAULT_CONTEXT = 'authentication'.freeze
```

Comment:
Do we still need this?

```suggestion
```

Reply:
Good catch. I think this can be safely removed.